### PR TITLE
[www] switch from glamor to emotion

### LINKED ISSUE #8816: [www]: switch from glamor to emotion
## Summary

We should swap from glamor to emotion for our CSS in JS needs on [gatsbyjs.org](./www)

### Basic example

Not much to document, but essentially the `css` prop is fully supported in emotion with `gatsby-plugin-emotion` so this should be a relatively doable refactor!

### Motivation

emotion has better theming support, and tends to do everything glamor does (and more!). We also used it on gatsbyjs.com (migrating from glamor -> emotion) and quite liked it.

See #8799 for more info re: theming.

### PR DESCRIPTION
**Pull request by janczizikow**
Fixes #8816

- Added emotion, it's gatsby plugin and related packages (emotion, emotion, sever, react-emotion) 
- Replaced keyframes with emotion syntax
- Replaced `css.inject()` with `injectGlobal`
- Added an empty string inside all the css styles with with `content:`` (otherwise emotion won't pick it up when using inline css)
- Removed glamor and gatsby glamor plugin from www dependencies
- Had fun contributing to gatsby website

I've tested everything locally everything seems to work fine.

Thanks to @DSchau for the tip regarding `style()`

### CODE REVIEW

**Comment by fk** [comment]
Hey @janczizikow! Thanks for taking this on! 🙏🤗

For some reason your review request went past me (probably because I was/am a little sick … I even took a shot myself just yesterday in #9045 😂); sorry for that (and thanks to @LeKoArts for the heads up)!

Will review ASAP!

File: www/src/components/email-capture-form.js [hunk 1]
```diff
@@ -3,7 +3,6 @@ import { rhythm, options } from "../utils/typography"
 import presets, { colors } from "../utils/presets"
```
Comment (fk) [inline www/src/components/email-capture-form.js]: While we're at this, could you remove the other unused imports here? 🙏 
See https://github.com/gatsbyjs/gatsby/pull/9045/files#diff-0b944fc95fef100bc9dd4aa6fbcb4647

File: www/src/components/hubspot-form.js [hunk 2]
```diff
@@ -57,7 +59,7 @@ export default class GatsbyHubspotForm extends Component {
           formId={formId}
           sfdcCampaignId={sfdcCampaignId}
           loading="Loading..."
-          css=""
+          {...{ css: `` }}
```
Comment (fk) [inline www/src/components/hubspot-form.js]: 👍 Nice catch!

File: www/src/components/plugin-searchbar-body.js [hunk 3]
```diff
@@ -18,7 +18,7 @@ import unescape from "lodash/unescape"
 import presets from "../utils/presets"
 import typography, { rhythm, scale } from "../utils/typography"
 import { scrollbarStyles } from "../utils/styles"
-import { css as glam } from "glamor"
+import { injectGlobal } from "emotion"
```
Comment (fk) [inline www/src/components/plugin-searchbar-body.js]: Hmm, wondering if we should always import from [react-emotion](https://emotion.sh/docs/react-emotion)?

/cc @DSchau

Comment (LeKoArts) [reply www/src/components/plugin-searchbar-body.js]: You can savely use react-emotion everywhere as emotion is included with that

Comment (fk) [reply www/src/components/plugin-searchbar-body.js]: 👍 ☝️ @janczizikow Could you change those imports from `emotion` to `react-emotion` please? 🙏 

(Thanks @LeKoArts!)

Comment (DSchau) [reply www/src/components/plugin-searchbar-body.js]: What @LeKoArts said ☝️ react-emotion just exports everything from emotion!

Comment (janczizikow) [reply www/src/components/plugin-searchbar-body.js]: @fk Sure, no problem 🙂

**Comment by janczizikow** [comment]
@fk No worries, I can have a look at this over the weekend and update the PR 🙂

**Comment by fk** [comment]
@janczizikow I got time to do this today and would be happy to. It wouldn't take away from your PR/author credits … I guess my fingers are just itching a little to get this in ;-).

Happy to hold my horses though, your call!

**Comment by fk** [comment]
Love your website btw! 💜

**Comment by janczizikow** [comment]
Hey @fk if you're itching to get into this go ahead 👍

**Comment by fk** [comment]
😻 🙏

**Comment by fk** [comment]
Thanks for your excellent work, and patience, and letting me poke around here Jan!
Super appreciated! 🙏

**Pull request merged**